code/datums/helper_datums/butchering.dm

```diff
@@ -4,7 +4,7 @@
 //The mob's butchering_drops list is made from get_butchering_products() when it dies. Humans however get it when they're created.
 
 /datum/butchering_product
-	var/obj/item/result
+	var/obj/item/result //can also be a list of items
```

Comment:
Instead of intentionally doing a bad type assignment and bragging about it in a comment here, why not just override the crab claws subtype?